### PR TITLE
links['last'] page number fix + example with `links` for docs

### PR DESCRIPTION
**Pull request by devalv**
Please correct me if I'm wrong, but The links['last'] (#83 ) have an index 1 more than the actual values. 

For example: if there are 10 records in the table - `total` will be 10, size by default = 50, `ceil (total / size)` will be `1` and the `last` value will be `1`, although it will be empty because real last page - `0`.

PS. Huge thanks for your awesome project! Maybe an additional example with a new feature to the doc won't be useless.

### CODE REVIEW

**Comment by uriyyo** [comment]
Thanks for your PR.
Could you please update tests to match new logic?

File: docs/customization.md [hunk 1]
```diff
@@ -38,6 +38,65 @@ async def route():
...
+            page, size, total = [value[k] for k in ("page", "size", "total")]
+
+            value["links"] = create_links(
+                first={"page": 0},
+                last={"page": ceil(total / size) - 1},
+                next={"page": page + 1} if (page + 1) * size < total else None,
+                prev={"page": page - 1} if 0 <= page - 1 else None,
+            )
+
+        return value
```
Comment (uriyyo) [inline docs/customization.md:97]:
```suggestion
from typing import TypeVar, Generic

from fastapi_pagination.links import Page

T = TypeVar("T")


class JsonApiPage(Page[T], Generic[T]):
    """JSON:API 1.0 specification says that result key should be a `data`."""

    class Config:
        fields = {"items": {"alias": "data"}}
```

**Review by uriyyo — CHANGES_REQUESTED** [review]
LGTM💫

I have one little suggestion on how your `JsonApiPage` example can be simplified using pydantic aliases.

File: docs/customization.md (hunk 1 above)
Comment (devalv) [reply docs/customization.md:97]: Wow! Looks great!

**Comment by uriyyo** [comment]
Thanks)

**Pull request merged**